assets/src/story-embed-block/block/storyPlayer.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function StoryPlayer({ url, title, poster, width, height }) {
+  return (
+    <amp-story-player
+      style={{
+        width: width ? `${width}px` : undefined,
+        height: height ? `${height}px` : undefined,
+      }}
+    >
+      <a
+        href={url}
```

Comment:
Should this perhaps open in another tab?
Or alternatively, the whole content could be disabled for interaction. Thoughts?

Reply:
For the editor we have to do something, yeah. But for save/frontend this is the expected markup.

In the editor I think it‘s only an issue when the player hasn‘t loaded yet. After page reload the interaction should work fine, no?

Reply:
It looks like clicking on the block leaves the editor and opens the embedded story instead if a story is loaded. Probably it shouldn't leave the editor when clicking on a block, e.g. if I click on the block to select it, I'm redirected instead.

Reply:
The issue is that the player currently doesn't load properly in the editor (noted in the PR description + discussion above). Once it does, it would behave like on the frontend: clicking on the story would just tap throuhg pages, not navigate away.

Reply:
Oh, okay, now I see what you meant by "loading properly".

Hmm, okay. Yes, perhaps the thing is that it didn't load properly for me when opening an already saved post with embeds (or after reload). Will check again a bit later, didn't focus on that when trying it out.

Reply:
Marking this here as resolved as this will be a non-issue once the player loads properly in the editor.